### PR TITLE
TST: adjust a `coordinates` test to avoid hardcoding exact data source url

### PR DESCRIPTION
**Pull request by neutrinoceros**
### Description
Just noticed this one test failing in pre-deps jobs. I don't know what's the ultimate source of truth for what this url should be but let me try the naive update first.

- [ ] By checking this box, the PR author has requested that maintainers do **NOT** use the "Squash and Merge" button. Maintainers should respect this when possible; however, the final decision is at the discretion of the maintainer that merges the PR.

### CODE REVIEW

File: astropy/coordinates/tests/test_name_resolve.py
Comment (mhvk) [inline astropy/coordinates/tests/test_name_resolve.py:119]: Maybe we need to be agnostic to the exact web site - this might depend on where the request came from (since vizier really is based at CDS, the harvard one is a mirror).

Comment (neutrinoceros) [reply astropy/coordinates/tests/test_name_resolve.py:119]: yes, that's probably best. Let me simplify this regexp

File: astropy/coordinates/tests/test_name_resolve.py
Comment (eerovaher) [inline astropy/coordinates/tests/test_name_resolve.py:119]:
```suggestion
            "Unable to find coordinates for name 'm87h34hhh' using http"
```
Furthermore, there's no need to call `re.escape()` anymore.

**Review by eerovaher — CHANGES_REQUESTED** [review]

File: astropy/coordinates/tests/test_name_resolve.py
Comment (neutrinoceros) [reply astropy/coordinates/tests/test_name_resolve.py]: nice catch. Done !

**Review by eerovaher — APPROVED** [review]
This is dealing with a flaky remote test, so it looks like something we'd want to backport, but I'll let @mhvk have a final look to confirm.

**Comment by neutrinoceros** [comment]
btw @astrofrog just released astropy 6.1.1 a couple hours ago and said he was planning a 6.1.2 tomorrow (mostly because #16578 missed the cut) so I guess we shouldn't be using the 6.1.1 milestone anymore and open a 6.1.2 on instead ?

**Comment by eerovaher** [comment]
I'm not quite sure whose job it is to create new milestones and branches. Should @pllim do this or should it be done as a step in the release process?

File: astropy/coordinates/tests/test_name_resolve.py
Comment (mhvk) [inline astropy/coordinates/tests/test_name_resolve.py:117]: According to the comment above, we should check for "SNV" in the url, so somehting like `...http.*sesame/SNV?m87h34hhh`

Comment (neutrinoceros) [reply astropy/coordinates/tests/test_name_resolve.py]: done

**Review by mhvk — APPROVED** [review]
Looks all good. Do we need to rebase to get rid of the readthedocs failure?

**Comment by neutrinoceros** [comment]
yup. Also had a regexp bug (I typed `SVN` instead of `SNV`, possibly due to some very *very* old muscle memory resurgence). Fixed and rebased !

**Comment by mhvk** [comment]
Thanks, @neutrinoceros!

**Comment by pllim** [comment]
Re: https://github.com/astropy/astropy/pull/16577#issuecomment-2173802084

@mhvk , you mean you can't create the milestone? That's  weird. Do you want me to check your permissions?

**Comment by mhvk** [comment]
@pllim - yes, I tried creating the milestone but didn't seem able to. I was surprised since I know I can add labels. But obviously no big deal.

**Comment by pllim** [comment]
@mhvk , I have always thought new milestone and new label use the same permission scheme. You are in the core maintainers team, so I don't know what is going on here.

**Pull request merged**
